rootplace/forms.py

```diff
@@ -22,6 +23,16 @@ class MergeForm(forms.Form):
         queryset=User.objects.all(),
     )
 
+    def clean(self):
+        cleaned_data = super().clean()
+        user1 = cleaned_data.get("user1")
+        user2 = cleaned_data.get("user2")
+
+        if user1 and user2 and user1.id == user2.id:
```

Comment:
```suggestion
        if user1.id == user2.id:
```
les deux sont déjà `required=True`, donc c'est redondant de vérifier qu'ils existent.